documentation/modules/proc-installing-crds.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// assembly-evaluation.adoc
+
+[id='proc-install-crds-{context}']
+= Installing {ProductName}
+
+Install {ProductName} with the Custom Resource Definitions (CRDs) required for deployment.
+
+.Prerequisites
+
+* Installation requires a Kubernetes account with cluster admin credentials.
+
+.Procedure
+
+. Log in to the Kubernetes cluster with cluster admin privileges.
+
+. Modify the installation files according to the `kafka` namespace where you will install the {ProductName} Kafka Cluster Operator.
++
+NOTE: By default, the files work in the `myproject` namespace.
++
+* On Linux, use:
++
+
+[source, shell, subs=+quotes]
+----
+sed -i 's/namespace: .\*/namespace: kafka/' install/cluster-operator/*RoleBinding*.yaml
+----
++
+* On Mac, use:
++
+[source, shell, subs=+quotes]
+----
+sed -i '' 's/namespace: .\*/namespace: kafka/' install/cluster-operator/*RoleBinding*.yaml
+----
+. Deploy the CRDs and role-based access control (RBAC) resources to manage the CRDs.
++
+[source, shell, subs=+quotes ]
+----
+kubectl create ns kafka
+kubectl apply -f install/cluster-operator/ -n kafka
+----
+
+. Create the project `my-kafka-project` where you will deploy your Kafka cluster.
++
+[source, shell, subs=+quotes ]
+----
+kubectl create ns my-kafka-project
+----
+
+. Enable the Cluster Operator to watch that namespace.
++
+--
+[source, shell, subs=+quotes]
+----
+kubectl edit deployment strimzi-cluster-operator STRIMZI_NAMESPACE=kafka,my-kafka-project -n kafka
```

Comment:
it's not needed having the CO watching its own `kafka` namespace.

Reply:
Right. I'll remove the command, but retain the commands to install the RoleBindings.

Reply:
I don't think `kubectl edit` understands an argument like `STRIMZI_NAMESPACE=kafka,my-kafka-project`. You probably need to explain what the user is going to edit. Or you could try to use a shell pipeline, though it might be difficult to select the right env var.